### ISSUE TITLE
Shadow glitches

### ISSUE THREAD

**Issue by sevn10** [SachaOropeza/AH64D-Project#92]
Depending on time of day the shadows in the cockpit get all glitched out. 

![20201120002039_1](https://user-images.githubusercontent.com/71986910/99762597-c2bff180-2aad-11eb-8e1a-d7f0974f82a0.jpg)

![20201120002112_1](https://user-images.githubusercontent.com/71986910/99762609-c81d3c00-2aad-11eb-9d25-9e52ef3d570c.jpg)

**Comment by SachaOropeza** [comment]
Interesting. 🤔

**Comment by sevn10** [comment]
Yeah, unfortunately it's not something I've been able to get consistently repeatable.
